### ISSUE TITLE
[1.14] Re-add White sprite to ModelLoader

### ISSUE THREAD

**Issue by Aeltumn** [MinecraftForge/MinecraftForge#6058]
1.12 had a really handy fully white sprite in the ModelLoader class, in 1.13/1.14 it got replaced with a dummy.

```java
// Temporary to compile things
    public static final class White {
        public static final ResourceLocation LOCATION = new ResourceLocation("white");
        public static final TextureAtlasSprite INSTANCE = MissingTextureSprite.func_217790_a();
    }
```
I can't find an issue about this and I don't know if anyone remembers that this needs re-adding so may this issue serve as a reminder.

**Comment by boq** [comment]
Original issue is #5555, proposed fix in #6032

**Comment by gigaherz** [comment]
Ah I wasn't aware of this issue. Note that with the new model loader now merged into both 1.14.4 and 1.15, you have "forge:white" available as an actual texture, which can be used by models or renderers. No need to wait for custom sprites to be reimplemented.
